app/models/miq_alert.rb

```diff
@@ -844,4 +845,23 @@ def self.import_from_yaml(fd)
       stat
     end
   end
+
+  def hawkular_keys
+    aux = super
```

Comment:
but what does aux stand for? the variable name is not clear (to me, so it's subjective)